shell/platform/android/android_context_vk_impeller.cc

```diff
@@ -67,6 +67,9 @@ static std::shared_ptr<impeller::Context> CreateImpellerContext(
       FML_LOG(IMPORTANT) << "Using the Impeller rendering backend (Vulkan).";
     }
   }
+  if (context->GetDriverInfo()->IsKnownBadDriver()) {
```

Comment:
Perhaps an INFO log saying this happened? Or will this cause issues with tests further down the line?

Reply:
How would that log be actionable?

Reply:
To the end user, it won't. But perhaps useful to know for us when we expect Vulkan but get OpenGL.

Reply:
Done